v2/api-validator/src/client/generated/models/CollateralDepositTransactionIntentRequest.ts

```diff
@@ -0,0 +1,15 @@
+/* generated using openapi-typescript-codegen -- do no edit */
+/* istanbul ignore file */
+/* tslint:disable */
+/* eslint-disable */
+
+import type { ApprovalRequest } from './ApprovalRequest';
+import type { CryptocurrencyReference } from './CryptocurrencyReference';
+import type { PositiveAmount } from './PositiveAmount';
+
+export type CollateralDepositTransactionIntentRequest = {
```

Comment:
We said we're currently not going to have an intent response per endpoint family